src/glang/lib/std/unicode.c3

```diff
@@ -354,6 +348,7 @@ function encode_to_utf8 : (scalar : UnicodeScalar, buffer: Span<u8>) -> View<u8>
 
     // Graphene cannot generate an invalid unicode scalar via `make<>` or the iterators
     // This is a bug in the user's code so I don't mind the unrecoverable error here
+    print(scalar.value);
```

Comment:
Leftover debug print!

Reply:
Noooo!!!! But also `runtime_assert` should take a string to print :upside_down_face:

Reply:
One day!